packages/expo-location/android/build.gradle

```diff
@@ -105,7 +105,7 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:${getKotlinVersion()}"
   }
 
-  api 'com.google.android.gms:play-services-location:20.0.0'
+  api 'com.google.android.gms:play-services-location:21.0.1'
```

Comment:
Do we know if it's okay to bump this now without affecting `react-native-maps`? we had to revert this [before](https://github.com/expo/expo/pull/23501)

Reply:
I didn't know that, good catch. I bumped it because location seemed to be working faster with the newer version. React-native-maps still uses `20.0.0` so I'll just revert it back to `20.0.0`.

Reply:
Ideally, we would bump it because then we can use the `LocationRequest.Builder` instead of `LocationRequest`. I bumped it to try to fix #20663 where there was a problem with location updates resolving the same promise twice. In theory, it shouldn't happen because we set `numUpdates` but on the older `LocationRequest` class this wasn't a guarantee. The `Builder` has `setMaxUpdate` which seems to fix the issue. I was hoping `react-native-maps` would have bumped their version by now 😏